### PR TITLE
make fetching URL-PMI optional (closes #33)

### LINKED ISSUE #33: Reference manifest spec
The spec says:

> In particular, owners of proprietary payment methods can use Web servers under their control to publish security information about authorized payment apps.

I think we should remove this. Identifiers should be identifiers - they are used by developers (for the purpose of the API) and should not be used in the manner above.

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/payment-method-id/dereferencing.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/payment-method-id/253f9e2...62ef24b.html)

### CODE REVIEW

**Comment by ianbjacobs** [comment]
Hi @marcoscaceres,

I am thinking we may want to be more explicit in cautionary statements to payment method owners. Here is the sort of flow I have in mind:

For user agents:
 * It is optional for user agents to fetch PMIs. (I think in your proposal you can drop " As <a>URL-based payment method identifiers</a> serve primarily as identifiers (and not a linked resource), ")

Note to payment method owners:
 * The Working Group is developing a payment method manifest specification that will give
    payment method owners greater control over the ecosystem of payment apps that implement
    their payment method.
* In the absence of a manifest or of broadly deployed implementations of one or more manifest specifications, there is no mechanism to limit which payment apps are able to claim to support a payment method.

[Hit "go" too soon before]

File: index.html [hunk 1]
```diff
@@ -125,6 +125,30 @@
...
+          identifiers (and not a linked resource), it is OPTIONAL for user
+          agents to <a data-cite="!WHATWG-FETCH#concept-fetch">fetch</a> a
+          <a>URL-based payment method identifier</a>.
+        </p>
+        <p class="note">
+          The working group is in the early stages of standardizing a <a href=
+          "https://w3c.github.io/payment-method-manifest/">Payment
+          Manifest</a>, which will allow developers to describe their payment
+          methods in a machine-processable format. We encourage developers to
+          experiment with Payment Manifest by placing one at the <a>URL-based
```
Comment (adrianhopebailie) [inline index.html]: This is incorrect. The manifest should be placed at a location that is accessed using the algorithm described at https://w3c.github.io/payment-method-manifest/#accessing

**Review by adrianhopebailie — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html]: Good catch, will add a reference to that.

Comment (marcoscaceres) [reply index.html]: I've decided to remove the note entirely. I don't feel comfortable referencing a spec that is not even a FPWD.

**Comment by marcoscaceres** [comment]
@ianbjacobs and @adrianhopebailie addressed your feedback. Please review.

**Review by adrianhopebailie — APPROVED** [review]

**Comment by adrianhopebailie** [comment]
My concern was an incorrect statement about the location of a manifest in relation to the URL-based PMI. That has been removed so no further comments from me.

**Review by ianbjacobs — COMMENTED** [review]
I'm ok with this change.

Marcos, please update the reference elsewhere in the spec from:
 https://w3c.github.io/webpayments/proposals/method-practice/

to:
 https://github.com/w3c/payment-request-info/wiki/PaymentMethodPractice

The WG supported moving the material to that repo.

Thank you,

Ian

**Pull request merged**